Tests/ParameterEncoderTests.swift

```diff
@@ -624,6 +624,20 @@ final class URLEncodedFormEncoderTests: BaseTestCase {
         XCTAssertEqual(result.success, "array=1&array=2")
     }
 
+    func testThatArraysCanBeEncodedWithCustomClosure() {
+        // Given
+        let encoder = URLEncodedFormEncoder(arrayEncoding: .custom({ key, index in
+          "\(key).\(index+1)"
```

Comment:
```suggestion
          "\(key).\(index + 1)"
```
A little spacing for the index.